test/test_files/function/gds/scc.test

```diff
@@ -0,0 +1,132 @@
+-DATASET CSV empty
+
+--
+
+-CASE Scc1
+-STATEMENT CREATE NODE TABLE User(id INT64, name STRING PRIMARY KEY);
+---- ok
+-STATEMENT CREATE REL TABLE FRIEND(FROM User to User);
+---- ok
+-STATEMENT CREATE (alice:User {id: 0, name: 'Alice'}),
```

Comment:
Your call but as a general principle, you could probably simplify these test data by giving them just generic and indexed names like U1 U2 etc. Probably would simplify writing down the graph for others who might read the code s you do below.